app/bundles/LeadBundle/Assets/css/lead.css

```diff
@@ -206,6 +206,37 @@ ul.tag-cloud li {
 .inline-spacer {
     text-transform: capitalize;
 }
+.contact{
+    display: inline !important;
+}
+.channel-disabled{
```

Comment:
Could this CSS be removed and use Bootstrap's `text-muted` instead? It seems to do the same thing. (http://getbootstrap.com/css/#helper-classes)